rio/src/RIO/Prelude/Reexports.hs

```diff
@@ -2,6 +2,12 @@
 module RIO.Prelude.Reexports
   ( module UnliftIO
   -- List imports from UnliftIO?
+  , UnliftIO.Concurrent.ThreadId
+  , UnliftIO.Concurrent.myThreadId
+  , UnliftIO.Concurrent.isCurrentThreadBound
+  , UnliftIO.Concurrent.yield
```

Comment:
This reexport in particular could be problematic, as it would conflict with streaming libraries (at least conduit). The others have quite unique namings.

Reply:
I see, maybe we can rename this to `yieldThread`?

Reply:
Sounds perfect!